tests/test_mars_format.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from unittest.case import TestCase
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import Label
+from datumaro.components.dataset import Dataset, DatasetItem
+from datumaro.components.environment import Environment
+from datumaro.plugins.mars_format import MarsImporter
+from datumaro.util.test_utils import compare_datasets
+
+from tests.requirements import Requirements, mark_requirement
+
+ASSETS_DIR = osp.join(osp.dirname(__file__), 'assets')
+DUMMY_MARS_DATASET = osp.join(ASSETS_DIR, 'mars_dataset')
+
+class MarsImporterTest(TestCase):
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_import(self):
+        expected_dataset = Dataset.from_iterable([
+            DatasetItem(id='0001C1T0001F001', image=np.ones((10, 10, 3)),
+                subset='train', annotations=[Label(label=2, attributes={
```

Comment:
Because each image in the dataset represents itself as clipped bounding box, and we don't have original image or information about coordinates of this bounding box in the original image.

Reply:
Probably, I don't quite understand the problem. We have an image, which is a bbox cut from the original image. But original images are not used in this format, so this bbox should just fit the whole image, shouldn't it?

Reply:
If I understand you correctly,  in such implementation we need to know size of image to assign the correct values to the coordinates of the bbox. Would there be an overhead of reading the image to find out the size of the image?

Reply:
It's a fair point. Yes, it can affect loading time, however, if we look at masks in VOC-like formats, it works pretty fine even with more complex processing. I think, we can try to implement this.